### PR TITLE
endangered language recognition espnet2 recipe

### PR DESCRIPTION
**Pull request by ftshijt**

### CODE REVIEW

File: egs2/TEMPLATE/asr1/path.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-MAIN_ROOT=$PWD/../../..
+MAIN_ROOT=/export/c04/jiatong/tools/espnet_update/espnet
```
Comment (sw005320) [inline egs2/TEMPLATE/asr1/path.sh:1]:
```suggestion
MAIN_ROOT= MAIN_ROOT=$PWD/../../..
```

File: egs2/puebla_nahuatl/asr1/path.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+../../open_li52/asr1/path.sh
```
Comment (sw005320) [inline egs2/puebla_nahuatl/asr1/path.sh:1]: It’s better to have an actual file instead of a link.

File: egs2/puebla_nahuatl/asr1/run.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+train_set=train
+train_dev=dev
+test_set=test
+
+asr_config=conf/train_asr.yaml
+inference_config=conf/decode_asr.yaml
+
+./asr.sh \
+    --local_data_opts "--stage 1" \
+    --stage 10 \
```
Comment (sw005320) [inline egs2/puebla_nahuatl/asr1/run.sh:17]: Please remove your own experimental options.

File: egs2/yoloxochitl_mixtec/asr1/path.sh [hunk 4]
```diff
@@ -0,0 +1 @@
+../../open_li52/asr1/path.sh
```
Comment (sw005320) [inline egs2/yoloxochitl_mixtec/asr1/path.sh:1]: Ditto

**Review by sw005320 — APPROVED** [review]
LGTM.
Once you fix the CI error, you can marge it.

**Comment by sw005320** [comment]
Many thanks!
This is very cool!

File: egs2/TEMPLATE/asr1/asr.sh [hunk 5]
```diff
@@ -93,7 +93,7 @@ inference_config= # Config for decoding.
 inference_args=   # Arguments for decoding, e.g., "--lm_weight 0.1".
                   # Note that it will overwrite args in inference config.
 inference_lm=valid.loss.ave.pth       # Language modle path for decoding.
-inference_asr_model=valid.acc.ave.pth # ASR model path for decoding.
+inference_asr_model=valid.acc.best.pth # ASR model path for decoding.
```
Comment (kamo-naoyuki) [inline egs2/TEMPLATE/asr1/asr.sh:96]: Why did you touch this value? This changing affects all previous recipes...

Comment (ftshijt) [reply egs2/TEMPLATE/asr1/asr.sh:96]: My apologies! This should not be changed. I'm sorry for that, could you please change it back?

Comment (kamo-naoyuki) [reply egs2/TEMPLATE/asr1/asr.sh:96]: Okay!

**Pull request merged**